tests/monitoring/vm_monitoring.go

```diff
@@ -86,7 +86,7 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 		}
 
 		BeforeEach(func() {
-			vmi := tests.NewRandomVMI()
+			vmi := libvmi.New(libvmi.WithResourceMemory("1Mi")
 			vm = libvmi.NewVirtualMachine(vmi)
```

Comment:
This will produce a failure in the subsequent `Create` since `libvmi.NewVirtualMachine(vmi)` will take the vmi namespace (which is nil).
In order to do this you should also change the 
```
virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm)
```
to 
```
virtClient.VirtualMachine(tests.GetTestNamespace(vm)).Create(context.Background(), vm)
```
Thank you!